modules/gitops-argocd-cli-logging-in-to-argocd-server.adoc

```diff
@@ -0,0 +1,76 @@
+// Module is included in the following assemblies:
+//
+// * gitops_cli_argocd/logging-in-to-argocd-server.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-argocd-cli-logging-in-to-argocd-server{context}"]
+= Logging in to the Argo CD server
+
+After you install and configure the {gitops-shortname} `argocd` CLI, to execute commands in the default mode, you must log in to the Argo CD server.
+
+.Prerequisites
+
+* You must have the {gitops-shortname} `argocd` CLI tool installed and configured.
+
+.Procedure
+
+Use your credentials to log in and remain logged in throughout the session. 
```

Comment:
Is this part of the procedure? I feel like we could use a discreet step here for logging in to the session.